src/GuiBase/Viewer/Viewer.cpp

```diff
@@ -60,7 +60,9 @@ KeyMappingViewer;
 #undef KMA_VALUE
 
 void Gui::Viewer::registerKeyMapping() {
-    m_keyMappingContext = Gui::KeyMappingManager::getInstance()->getContext( "ViewerContext" );
+    auto kMapMan = Gui::KeyMappingManager::getInstance();
```

Comment:
not so important : kMapMan -> keyMappingManager or keyMapping or keyMap

Reply:
done